fhir-examples-generator/src/main/java/com/ibm/fhir/examples/DataCreatorBase.java

```diff
@@ -237,6 +239,16 @@ protected Object createArgument(Class<?> owningClass, Method builderMethod, Clas
         return builder.extension(e);
     }
 
+    protected CodeableConcept.Builder setDataAbsentReasonCoding(CodeableConcept.Builder builder) {
+        Coding coding = Coding.builder()
+                .extension(Extension.builder()
+                .url("http://hl7.org/fhir/StructureDefinition/data-absent-reason")
+                .value(Code.of("unknown"))
+                .build())
+                .build();
```

Comment:
because our object are immutable, you could actually "cheat" and re-use the same one in multiple places.
still, thats an optimization that probably isn't needed in the example generator, just wanted to make you aware of the possibility

Reply:
thanks for the tip!